src/main/java/com/vinylteam/vinyl/web/controller/SignupController.java

```diff
@@ -19,56 +20,58 @@
 @RestController
 @Slf4j
 @RequiredArgsConstructor
-@CrossOrigin(origins = {"http://localhost:3000", "https://react-wax-deals.herokuapp.com"})
 public class SignupController {
```

Comment:
Why do we need to return Map<String, Object> and not Map<String, String> ?
As far as I am concerned you always return Map<String, String> but have to wrap it due to the methods return type
Why we cannot just return Map<String, String> from controller methods?

Feel free to call me to discuss